programs/create-genesis/genesis_create.cpp

```diff
@@ -198,6 +205,12 @@ struct state_object_visitor {
         early_exit = true;
     }
 
+    void operator()(const golos::withdraw_vesting_route_object& w) {
+        if (w.from_account != w.to_account && w.percent > 50 * config::percent_1) {
```

Comment:
Some strange value in 50%. It will be difficult to explain to the community why, with a route of 60%, another account gets the remaining 40%. Since almost all routes are either less than 50% or equal to 100%, it is proposed to use a 100% border. Otherwise, the funds are displayed on the owner’s account, which can then transfer a certain amount in the form of tokens to other accounts.